ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageFragment.kt

```diff
@@ -98,4 +105,15 @@ class PreviewImageFragment : Fragment() {
             }
         )
     }
+
+    private fun startUCrop(inputFilePath: String) {
+        findNavController().navigate(
+            R.id.action_previewFragment_to_ucropFragment,
+            UCropUtil.getUCropOptionsBundle(
+                File(inputFilePath),
+                // TODO: output file location
+                File(requireContext().cacheDir, UCropUtil.IMAGE_EDITOR_OUTPUT_IMAGE_FILE_NAME)
```

Comment:
I think we might want to consider moving this little piece of logic into the VM. We could pass the cacheDir into `vm.onCreateView` and pass both `inputFilePath` and `outputFilePath` into the LiveData. Wdyt?

>UCropUtil.IMAGE_EDITOR_OUTPUT_IMAGE_FILE_NAME

How would you feel about moving this constant into the VM? I feel like it's too specific to live in a utility class, wdyt?

Reply:
Refactored here 7552d63 and 87fb32a